### PR TITLE
Add getenv to template function map

### PR DESCRIPTION
**Pull request by SeBBBe**
**Checklist**

- [x] tests added
- [x] commits conform to the [Commit message format](https://github.com/qlik-oss/gopherciser/blob/master/.github/CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

Add getenv to template function map.

### CODE REVIEW

File: docs/settingup.md [hunk 1]
```diff
@@ -2379,6 +2380,21 @@ The following functions are supported:
     }
 }
 
+```
+
+```json
+{
+    "action": "ElasticCreateApp",
```
Comment (DnlLrssn) [inline docs/settingup.md:2387]: please use non elastic action as example, have a branch removing all mention of elastic actions from docs and don't want new ones.

File: generatedocs/data/extra/sessionvariables/description.md [hunk 2]
```diff
@@ -56,5 +57,20 @@ The following functions are supported:
     }
 }
 
+```
+
+```json
+{
+    "action": "ElasticCreateApp",
```
Comment (DnlLrssn) [inline generatedocs/data/extra/sessionvariables/description.md:64]: same here please use QSEoW action.

File: generatedocs/pkg/genmd/testdata/settingup.md [hunk 3]
```diff
@@ -2068,6 +2069,21 @@ The following functions are supported:
...
 }
 
+```
+
+```json
+{
+    "action": "ElasticCreateApp",
+    "label": "Create new app",
+    "settings": {
+        "title": "{{env \"TITLE\"}}",
```
Comment (DnlLrssn) [inline generatedocs/pkg/genmd/testdata/settingup.md:2079]: same

**Comment by DnlLrssn** [comment]
code lgtm though

**Comment by SeBBBe** [comment]
Alright. I realize I updated some files I didn't have to, but it doesn't really matter. All working now!

**Review by DnlLrssn — APPROVED** [review]

**Pull request merged**